spec/spec_helper_system.rb

```diff
@@ -0,0 +1,27 @@
+require 'rspec-system/spec_helper'
```

Comment:
I believe ```spec_helper_system.rb``` is no longer used. Nowadays we only use ```spec_helper_acceptance.rb```

Reply:
I'll get that as well.

Reply:
Done.